index.html

```diff
@@ -0,0 +1,1005 @@
+            var c = d.createElement('script');
+            o.api = new Array();
+            c.async = true;
+            c.type = 'text/javascript';
+            c.charset = 'utf-8';
+            c.src = 'https://web-sdk.smartlook.com/recorder.js';
+            h.appendChild(c);
+        })(document);
+        smartlook('init', '3d8cc3fad36fd64fa8461c440815b246c3b75961', {
+            region: 'eu'
+        });
+    </script>
+</head>
+
+<body>
+    <nav class="container-fluid">
+        <div class="container-fluid">
+            <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
+                <a href="/" class="d-flex align-items-center mb-2 mb-lg-0 text-white text-decoration-none">
+                    <div class="logo">logo</div>
+                </a>
+                <a class="navbar-brand" href="#">Navbar</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <ul class="nav col-12 col-lg-auto me-lg-auto mb-2 justify-content-center mb-md-0">
+                    <li><a href="#" class="nav-link px-2 text-secondary">Features</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Pricing</a></li>
+                    <li><a href="#" class="nav-link px-2 text-white">Industries</a></li>
+                </ul>
+                <!--
+                <form class="col-12 col-lg-auto mb-3 mb-lg-0 me-lg-3" role="search">
+                <input type="search" class="form-control form-control-dark text-bg-dark" placeholder="Search..." aria-label="Search">
+                </form>-->
+
+                <div class="text-end">
+                    <!--<button type="button" class="btn btn-outline-light me-2">Login</button>-->
+                    <button type="button" class="btn btn-info">Request a meeting</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+
+    <header class="container-fluid message">
+        <div class="row">
+            <div class="col-12 text-center">
+                <p><i><b>FlyArrow.io</b></i> is a project of the <b>ARROW DAO</b>, a non-profit decentralized organization building community owned aerial solutions. <a href="https://www.arrowair.com" target="_blank">Visit the DAO website</a></p>
+            </div>
+        </div>
+    </header>
+
+    <div class="container-fluid bg-main text-light">
+        <div class="row mb-5">
+            <div class="col-6 pe-5">
+                <h1>Building VTOL software for automated scheduled flights</h1>
+                <p>With our full-stack flightplan system you can take the pain out of launching a safe and fast system to organize and monetize on your VTOL assets and services.</p>
```

Comment:
This can be more prominent than just a <p> tag.

Reply:
fixed: b836fc5bb3bb5beca3fbc976432ab58088de0d6b